src/ocm/components/clusterConfiguration/OcmClusterDetailsFormFields.tsx

```diff
@@ -68,6 +69,7 @@ export const OcmClusterDetailsFormFields = ({
     nameInputRef.current?.focus();
   }, []);
 
+  const isSingleClusterFeatureEnabled = useFeature('ASSISTED_INSTALLER_SINGLE_CLUSTER_FEATURE');
```

Comment:
The only thing I can think of to make the UI more flexible is not to use directly `useFeature`, but instead create a similar function which can tell us which UI features are enabled or not. 

For example
const { hasNetworkConfigurationControlGroup } = getUiFeatureConfig();

This new function would centralise the logic about what UI features are enabled in each scenario. 
If it only depends on data we have on the cluster, it can be feasible.

Reply:
Thank you. Let's take this offline. I understand your point but it seems to me like wrapping a wrapper.
It's subjective if that brings value, I am afraid that this is the a parallel to having 5 opinions per 5 people asked.